.gitlab/common/shared.yml

```diff
@@ -13,28 +13,6 @@
   - $ARTIFACTORY_BYPASS=$(& "$env:CI_PROJECT_DIR\tools\ci\aws_ssm_get_wrapper.ps1" $ARTIFACTORY_BYPASS_SSM_NAME)
   - if ($Env:ARTIFACTORY_BYPASS -eq "true") { Write-Host "Bypassing Artifactory" }
 
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I believe we should remove the code above as well

Reply:
Thanks for the catch! I somehow missed this reference while grepping.